src/input.ts

```diff
@@ -80,14 +80,14 @@ export class LeanInputExplanationHover implements HoverProvider, Disposable {
 }
 /* Each editor has their own abbreviation handler. */
 class TextEditorAbbrevHandler {
-    range: Range | undefined;
+    ranges: (Range | undefined)[] = [];
```

Comment:
```suggestion
    ranges: Range[] = [];
```

If I understand the code correctly.

Reply:
> This is very much intentional because it allows you to type `"\r\n"`, etc.

Ah OK, that makes sense - I'll have a go at reenabling that behaviour ☺️

Reply:
👍